src/EncumberableToken.sol

```diff
@@ -243,15 +243,15 @@ contract EncumberableToken is ERC20, IERC20Permit, IERC7246 {
         bytes32 r,
         bytes32 s
     ) external {
-        require(block.timestamp < expiry, "Signature expired");
+        require(block.timestamp < expiry, "ERC7246: signature expired");
```

Comment:
These are more of a ERC712 error though. I'm not sure if it makes sense to prepend these ones.

Reply:
agreed; removed the changes related to the bySig functions